### ISSUE TITLE
Scraping this project and adding the VM approach to Calamares instead

### ISSUE THREAD

**Issue by tgharib** [manjaro-arm/manjaro-arm-installer-cli#7]
After speaking to Kevin on #calamares on freenode, we've realized that it'd be easier to just add the VM approach to Calamares then recreating an installer from scratch.

It seems the majority of ARM devices are IoT devices so installing the OS from a live media or using in-place mode of Calamares is not an option. As a result, the VM approach is required on these devices.

**Comment by dodgejcr** [comment]
Are we 100% that we want to do carry around the full calamares for this? The only reservations to this I can see is users having to install a full calamares onto their machines. I dont know the power of calamares in the in-place mode but hopefully users cant accidentally mess with their x86 installs by using this.

**Comment by gryffyn** [comment]
I agree. live media is not an option for ARM devices. I think dodge is right. If users can't screw up their x86, then I love the idea.

**Comment by tgharib** [comment]
That's a good point about messing up their x86 installs. Maybe we should have a fork that specifically disables x86 installs so that there's no risk there.

On another note, I think the main benefit to forking Calamares is the included partition editing and the nice/uniform UI. The partition editing seems like it would be a pain to do correctly (probably not something I would want to do; never mind the risk of messing up). We can have the user edit partitions outside of the installer but then it would feel a tad half-arsed.
